Cargo.toml

```diff
@@ -3,13 +3,13 @@ name = "kakarot-rpc"
 version = "0.1.0"
 edition = "2021"
 authors = [
-  "Abdelhamid Bakhta <@abdelhamidbakhta>",
-  "Elias Tazartes <@eikix>",
-  "Clement Walter <@clementwalter>",
-  "Gregory Edison <@greged93>",
-  "Johann Bestowrous <@jobez>",
-  "Harsh Bajpai <@bajpai244>",
-  "Danilo Kim <@danilowhk>",
+    "Abdelhamid Bakhta <@abdelhamidbakhta>",
```

Comment:
the current formatter is trunk in the team's IDE, not sure why it's not defaulting to the same cargo fmt config though

Reply:
i think thats because i was not using vscode, will try using it 👍

Reply:
still the same when using vscode. not sure whats the issue.